jonah-dev/lfd2-bot
# Adds requirements.txt and virtualenv

## Summary
The goal of this PR is to explicitly provide Python requirements via a **requirements.txt** file to make this project's environment replicable for any developer.

## Testing Instructions
1. Follow the instructions added in **README.md** to create a new virtual environment with all required dependencies in your host.
2. Confirm you are able to execute the bot commands outlined in it's help command. **bot.py** lists the following, though there are other optional commands as well:
```python
?start - Initializes the lobby
?join - Joins the lobby
?leave - Leaves the lobby
?lobby - View the lobby
?order66 - Issues a ping to begin the game
?lag - Calculates your current lag
?ped - FOOTBALL SZN
?ready - Ready up!
?unready - Not R!
```
